src/toil/utils/toilRsyncCluster.py

```diff
@@ -0,0 +1,47 @@
+# Copyright (C) 2015 UCSC Computational Genomics Lab
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""
+SSHs into the toil appliance container running on the leader of the cluster
+"""
+import logging
+from toil import version
+from toil.lib.bioio import getBasicOptionParser, parseBasicOptions, setLoggingFromOptions
+
+logger = logging.getLogger(__name__)
+
+
+def main():
+    parser = getBasicOptionParser()
+    parser.add_argument("--version", action='version', version=version)
+    parser.add_argument('-p', "--provisioner", dest='provisioner', choices=['aws'], required=True,
```

Comment:
It did when I started working on it - it was refactored in 1e0febf and I didn't notice. I'll update it to match .toilSSHCluster as it is now.